src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -110,6 +110,12 @@ private TestKitBase(ITestKitAssertions assertions, ActorSystem system, Config co
             {
                 InternalCurrentActorCellKeeper.Current = (ActorCell)((ActorRefWithCell)testActor).Underlying;
             }
+            else if(!(this is TestProbe)) 
```

Comment:
This is obviously ugly in every possible way. but the testkit currently requires two different behaviors based on type.
(it could be extracted to a virt method, but then again, virt methods from ctors and all that.)